packages/python/accern_xyme/types.py

```diff
@@ -462,3 +464,12 @@ class DagDef(BaseDagDef):
 DeleteBlobResponse = TypedDict('DeleteBlobResponse', {
     "blobs": List[DeleteBlobStatus],
 })
+BlobDetails = TypedDict('BlobDetails', {
+    "blob_id": str,
```

Comment:
in TS?

Reply:
There are required mainly for upload functionality which is something we have skipped for typescript. Therefore these 2 types do not exist in TS